src/bls/bls.cpp

```diff
@@ -135,7 +139,11 @@ CBLSSignature CBLSSecretKey::Sign(const uint256& hash) const
 void CBLSPublicKey::AggregateInsecure(const CBLSPublicKey& o)
 {
     assert(IsValid() && o.IsValid());
-    impl = Scheme(bls::bls_legacy_scheme.load())->Aggregate({impl, o.impl});
+    try {
+        impl = Scheme(bls::bls_legacy_scheme.load())->Aggregate({impl, o.impl});
+    } catch (...) {
+
```

Comment:
shouldn't we do something instead of nothing here? or at least add a comment as to why nothing is okay?

Reply:
@PastaPastaPasta comment or log you mean?